app/src/main/java/com/mstudio/superheroarch/CharacterDto.kt

```diff
@@ -0,0 +1,24 @@
+package com.mstudio.superheroarch
+
+data class CharacterDto(
+    val name: String,
+    val status: String,
+    val image: String,
+    val origin: UbicationDto?,
+    val location: UbicationDto?,
+    val episode: List<String>
+)
+
+data class UbicationDto(
+    val name: String,
+    val url: String
```

Comment:
you are not using this url.
This makes you create a nested object, now you have mappers, you can remove the ubication object after parse it from remote and work with a simple originName: String and locationName: String.

Reply:
Fixed in next commit, the url variable has been removed